### DISCUSSION TITLE
_createnovalidate API error

### DISCUSSION THREAD

**Discussion by ankitbhansali** [egovernments/Digit-Core#131] [DIGIT Platform]
Currently, All pods are running fine in QUick Setup.
I can access the employee page.
But when i run the postman collection to create super user, i get the below response.

"Errors": [
{
"code": "BadRequest",
"message": "400 : [{"ResponseInfo":null,"Errors":[{"code":"InvalidAccessTokenException","message":"An unhandled exception occurred on the server","description":"Invalid Access Token Exception","params":null}]}]",
"description": "400 : [{"ResponseInfo":null,"Errors":[{"code":"InvalidAccessTokenException","message":"An unhandled exception occurred on the server","description":"Invalid Access Token Exception","params":null}]}]",
"params": null
}
]

Re

**Comment by talele08** [comment]
You need to port-forward to create the root user. Please use the command:
kubectl port-forward USER_SERVICE_PODNAME 8080:8080

You can then call the API with host: localhost:8080

**Reply by ankitbhansali** [reply]
Thank you. Root user is now created.
But still facing issues in other APIs
How about other APIs, for example: http://localhost:8080/localization/messages/v1/_upsert
When i port forward "Localization" pod, i get user authentication failed.
If i dont forward "Localization" then i get 404 not found
